internal/store/deployment.go

```diff
@@ -41,6 +41,15 @@ var (
 	descDeploymentLabelsDefaultLabels = []string{"namespace", "deployment"}
 )
 
+var (
+	allowedDeploymentReasons = map[string]struct{}{
+		"MinimumReplicasAvailable": {},
```

Comment:
Looking at these strings again
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/util/deployment_util.go#L67

Can we refer to these here instead and create a complete list based on what's available in deployment_util.go?
```suggestion
import deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
var (
	allowedDeploymentReasons = map[string]struct{}{
		deploymentutil.MinimumReplicasAvailable: {},
```

Reply:
should I also remove the empty string too then?

Reply:
When I'm trying to:
```bash
go get k8s.io/kubernetes/pkg/controller/deployment/util
```

I'm getting this, so should I directly use strings?

Reply:
Good point, you're right!

Can you copy all other reasons over as a string and add a comment that they are taken from deployment_utils.go?